daffodil-lib/src/main/scala/edu/illinois/ncsa/daffodil/schema/annotation/props/PropertyScoping.scala

```diff
@@ -209,8 +209,10 @@ trait FindPropertyMixin extends PropTypes {
    * For unit testing convenience
    */
   final def verifyPropValue(key: String, value: String): Boolean = {
-    findPropertyOption(key) match {
-      case Found(`value`, _, _, _) => true
+    val prop = findPropertyOption(key)
+    val valueLC = value.toLowerCase
+    prop match {
+      case Found(v, _, _, _) if (v.toLowerCase == valueLC) => true
```

Comment:
I was about to make a comment about using String.equalsIgnoreCase since that might be a little faster in some cases. But then realized this is only used in unit tests. Should this just be moved to a unit test file? Also seems that we would want to be strict about checking case during unit tests? Case matters with property values, right?

Reply:
Removed case insensitivity. Only one test had to be changed to UTF-8 from "utf-8". 

Moving this is problematic, as tests depend on this being available on DSOM objects so as to examine specific object for properties. Might be convenient to have when debugging as well. So I'll leave it on FindPropertyMixin for now.